src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -194,7 +180,6 @@
 
 (handlers/register-handler-fx
  :check-transactions-queue
- [(re-frame/inject-cofx :now)]
```

Comment:
`:now` coeffect is injected into default interceptor chain, that's why there is no need for this.